bindings/bindings-linux-aarch64-R4.2.rs

```diff
@@ -624,42 +645,54 @@ pub type R_altstring_Is_sorted_method_t =
-pub const GEUnit_GE_CM: GEUnit = 3;
+#[repr(u32)]
 #[doc = "The graphics engine will only accept locations and dimensions\n in native device coordinates, but it provides the following functions\n for converting between a couple of simple alternative coordinate\n systems and device coordinates:\n    DEVICE = native units of the device\n    NDC = Normalised device coordinates\n    INCHES = inches (!)\n    CM = centimetres (!!)"]
-pub type GEUnit = u32;
-#[doc = "In response to this event, the registered graphics system\n should allocate and initialise the systemSpecific structure\n\n Should return R_NilValue on failure so that engine\n can tidy up memory allocation"]
-pub const GEevent_GE_InitState: GEevent = 0;
-#[doc = "This event gives the registered system a chance to undo\n anything done in the initialisation."]
-pub const GEevent_GE_FinaliseState: GEevent = 1;
-#[doc = "This is sent by the graphics engine prior to initialising\n the display list.  It give the graphics system the chance\n to squirrel away information it will need for redrawing the\n the display list"]
-pub const GEevent_GE_SaveState: GEevent = 2;
-#[doc = "This is sent by the graphics engine prior to replaying the\n display list.  It gives the graphics system the chance to\n restore any information it saved on the GE_SaveState event"]
-pub const GEevent_GE_RestoreState: GEevent = 6;
-#[doc = "Copy system state information to the current device.\n This is used when copying graphics from one device to another\n so all the graphics system needs to do is to copy across\n the bits required for the display list to draw faithfully\n on the new device."]
-pub const GEevent_GE_CopyState: GEevent = 3;
-#[doc = "Create a snapshot of the system state that is sufficient\n for the current \"image\" to be reproduced"]
-pub const GEevent_GE_SaveSnapshotState: GEevent = 4;
-#[doc = "Restore the system state that is saved by GE_SaveSnapshotState"]
-pub const GEevent_GE_RestoreSnapshotState: GEevent = 5;
-#[doc = "When replaying the display list, the graphics engine\n checks, after each replayed action, that the action\n produced valid output.  This is the graphics system's\n chance to say that the output is crap (in which case the\n graphics engine will abort the display list replay)."]
-pub const GEevent_GE_CheckPlot: GEevent = 7;
-#[doc = "The device wants to scale the current pointsize\n (for scaling an image)\n This is not a nice general solution, but a quick fix for\n the Windows device."]
-pub const GEevent_GE_ScalePS: GEevent = 8;
-pub type GEevent = u32;
-pub const R_GE_lineend_GE_ROUND_CAP: R_GE_lineend = 1;
-pub const R_GE_lineend_GE_BUTT_CAP: R_GE_lineend = 2;
-pub const R_GE_lineend_GE_SQUARE_CAP: R_GE_lineend = 3;
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum GEUnit {
+    #[doc = "native device coordinates (rasters)"]
+    GE_DEVICE = 0,
+    #[doc = "normalised device coordinates x=(0,1), y=(0,1)"]
+    GE_NDC = 1,
+    GE_INCHES = 2,
+    GE_CM = 3,
```

Comment:
<s>I think we lost docstrings here, no?</s> My bad, did not notice rearrangement.